common/src/chain/transaction/utxo_outpoint.rs

```diff
@@ -105,6 +93,38 @@ impl UtxoOutPoint {
     }
 }
 
+impl rpc_description::HasValueHint for UtxoOutPoint {
+    const HINT_SER: rpc_description::ValueHint = RpcUtxoOutPoint::HINT_SER;
+    const HINT_DE: rpc_description::ValueHint = RpcUtxoOutPoint::HINT_DE;
+}
+
+#[derive(Debug, Clone, serde::Serialize, serde::Deserialize, rpc_description::HasValueHint)]
+#[serde(tag = "type", content = "content")]
+enum RpcUtxoOutPoint {
```

Comment:
`RpcUtxoOutpoint` exists in `chainstate/src/rpc/types/input.rs`

Reply:
These are slightly different. The chainstate one is being directly used by the wallet. The common one is a private implementation detail for serialize/deserailize of UtxoOutPoint (which exists for the same reason I explained in the other comment).

Reply:
Will try to at least unify the underlying implementation.

Reply:
Hmm it requires me break RPC backwards compatibility in existing calls like `get_utxo`. The hope was this would only affect the token stuff.

Reply:
It's a more general problem. In order to make the enum representation consistent accross the board, we also need to change `OutPointSourceId`. That, however, means changing stuff that's likely already used like coin selection in wallet rpc or `get_utxo`. It does not apply only to the token-related stuff.